client/src/pages/Brofile.js

```diff
@@ -2,10 +2,11 @@ import { Route, Link } from 'react-router-dom'
 import styled from 'styled-components/macro';
 import profilepic from '../images/profilepic.svg'
 /*import EventButton from '../components/EventButton';*/
-import tagLogo from '../icons/edittags.svg';
-import editbrofileicon from '../icons/editbrofileicon.svg';
-import EventTags from '../components/EventTags';
-
+/*import tagLogo from '../icons/edittags.svg';*/
+/*import editbrofileicon from '../icons/editbrofileicon.svg';*/
+/*import EventTags from '../components/EventTags';*/
```

Comment:
Do you need these imports? if not its better to delete them. this will keep your project clean